packages/vue3/src/components/Table/Table.vue

```diff
@@ -220,6 +220,7 @@ function selectRow(index: number, data: boolean) {
   align-items: center;
   justify-content: center;
```

Comment:
```suggestion
  justify-content: start;
```
This should fix the table position being centered rather than at the beginning
![Screenshot 2023-12-08 at 3 22 03 PM](https://github.com/youcan-shop/youcan-ui/assets/143792300/9729cab9-f7c2-4959-a55f-c7982b275256)